lib/pages/openedListPage.dart

```diff
@@ -0,0 +1,310 @@
+import 'package:MobileOne/localization/localization.dart';
+import 'package:flutter/material.dart';
+import '../localization/localization.dart';
+
+const Color GREEN = Colors.green;
+const Color GREY = Colors.grey;
+const Color GREY600 = Colors.grey;
+const Color RED = Colors.red;
+const Color WHITE = Colors.white;
+const Color TRANSPARENT = Colors.transparent;
+
+class OpenedListPage extends StatefulWidget {
+  OpenedListPage({Key key}) : super(key: key);
+
+  @override
+  OpenedListPageState createState() => OpenedListPageState();
+}
+
+class OpenedListPageState extends State<OpenedListPage> {
+  final List<Widget> widgetList = [];
+  final _itemNameController = new TextEditingController();
+  final _itemCountController = new TextEditingController();
+  String _itemName;
+  int _itemCount = 0;
+  String _itemType;
+
+  @override
+  void dispose() {
+    _itemNameController.dispose();
+    _itemCountController.dispose();
+    super.dispose();
+  }
+
+  void handleSubmittedItemCount(int input) {
+    _itemCount = input;
+  }
+
+  void handleSubmittedItemName(String input) {
+    _itemName = input;
+  }
+
+  addDatabaseItemsToList() {}
```

Comment:
why this empty function ?

Reply:
Because we don't get items from database yet